Gems/Atom/Asset/Shader/Code/Source/Editor/AzslShaderBuilderSystemComponent.cpp

```diff
@@ -83,7 +83,7 @@ namespace AZ
             // Register Shader Asset Builder
             AssetBuilderSDK::AssetBuilderDesc shaderAssetBuilderDescriptor;
             shaderAssetBuilderDescriptor.m_name = "Shader Asset Builder";
-            shaderAssetBuilderDescriptor.m_version = 124; // Add support for small SRGs
+            shaderAssetBuilderDescriptor.m_version = 125; // Add support for small SRGs
```

Comment:
I'm not sure if the comment is still valid, this is a new increment because of the constants, right?